app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] An error now occurs when a user tries to sign up. Please take a look at the error image attached below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/201360388-ed8022b3-d6c3-4974-834f-88ca84ef7232.png)